db/migrate/20190824200120_create_orders.rb

```diff
@@ -8,6 +8,7 @@ def change
       t.string :city
       t.string :state
       t.integer :zip
+      t.integer :status, default: 0
```

Comment:
Add column from the command line: rails g migration AddStatusToOrders status:integer
I believe a default value must still be added in the migration file, however.